src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,138 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String REALLY_STRONG_PASSWORD = "123456789";
+    private static final int REALLY_COOL_AGE = 30;
+    private static final User PREPARED_FOR_CHECKING_LOGIN_USER_1 =
```

Comment:
Bad name, this name does not have any context and says nothing about what data stored here